### DISCUSSION TITLE
How to integrate with hue when kyuubi language is scala rather than sql

### DISCUSSION THREAD

**Discussion by MyqueWooMiddo** [apache/kyuubi#5026] [Q&A]
I've already successfully integrated kyuubi with hue when language is sql by default.

But

How to integrate kyuubi with hue when language is scala rather than sql ?  I could not achieve with many attempts.

**Comment by HaoYang670** [comment]
I think the language can be changed at anytime as long as you set `kyuubi.operation.language`
